Dz3/User.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static DZ2DZ3.Program;
+using System.Runtime.Intrinsics.X86;
+using System.Net.Cache;
+using System.Runtime.CompilerServices;
+
+namespace DZ2DZ3
+{
+    public class User
+    {
+        private string firstName;
+        private string lastName;
+        private Gender gender;
+        private string a;
```

Comment:
What is that `a` means?

Reply:
this mean that anyone can`t change my code, coz my fields are private, are u asking that?